eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/handle/WebhookHttpSinkHandler.java

```diff
@@ -49,7 +49,7 @@ public class WebhookHttpSinkHandler extends CommonHttpSinkHandler {
     private final HttpWebhookConfig webhookConfig;
 
     // store the callback data
-    private final BlockingQueue<Object> callbackQueue;
+    private final BlockingQueue<JSONObject> callbackQueue;
```

Comment:
The naming of `callback data` and `callbackQueue` is a little ambiguous. Does it mean `received responses`?

Actually, the process of HTTP Sink Connector sending requests to callbackUrl is called "callback", and the process of HTTP Sink Connector receiving response from callbackUrl is just finishing a normal HTTP request. So we'd better clear these concepts.

Reply:
Okay, I'm going to rename it to `receivedDataQueue`